simple/single-script/create/funcs

```diff
@@ -214,7 +214,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```

Comment:
We can point this to the live site. WDYT?

Reply:
Yes, +1, have a look on the same for EI and IS too.

Reply:
Is it hosted yet?

Reply:
It should be hosted here https://wso2.com/products/install/faq/#Kubernetes. We put Simplified deployment related info to the FAQ  but it seems like it is not in the live yet.

Reply:
Now it's live https://wso2.com/products/install/faq/#Kubernetes